cmd/kubeadm/app/cmd/phases/util_test.go

```diff
@@ -44,8 +45,8 @@ func TestSetKubernetesVersion(t *testing.T) {
 		},
 		{
 			name:   "any other version is skipped",
-			input:  "v1.12.0",
-			output: "v1.12.0",
+			input:  constants.MinimumKubernetesVersion,
+			output: constants.MinimumKubernetesVersion,
```

Comment:
Again, we don't need this to test `SetKubernetesVersion`. It can stay forever on v1.12.0.